lib/pdf/collate.rb

```diff
@@ -64,7 +64,7 @@ def self.merge_pdfs(output_file, *filenames)
 
     unless status == 0
       puts output
-      exit status
+      raise "Ghostscript error: non-zero status of #{status}"
```

Comment:
Note that the only non-test [caller](https://github.com/code-dot-org/code-dot-org/blob/staging/pegasus/sites/virtual/collate_pdfs.rake#L22) [rescues and reraises exceptions](https://github.com/code-dot-org/code-dot-org/blob/staging/pegasus/sites/virtual/collate_pdfs.rake#L40-L46), so this shouldn't have an impact there.